project1.css

```diff
@@ -0,0 +1,599 @@
+header {
+  /* header-background-mobile */
+
+  position: absolute;
+  width: 375px;
+  height: 1618px;
+  left: 500px;
+  top: 0;
+
+  /* Neutrals/N0 */
+
+  background: #fff;
+}
+
+section {
+  /* Headline */
+
+  /* Auto layout */
+
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 114px 24px;
+  gap: 12px;
+  position: absolute;
+  width: 375px;
+  height: 1618px;
+  left: calc(50% - 375px / 2);
+  top: 92px;
+}
+
+.flex-container {
+  display: flex;
+  flex-direction: column;
+  position: absolute;
+  width: 375px;
+  height: 418px;
+  left: calc(50% - 375px / 2);
+  top: 552px;
+}
```

Comment:
- [ ] [REQUIRED] Good job so far! The design in `Works` and `About myself` section of your project is not exactly like [Figma design](https://www.figma.com/files/recent?fuid=1045426358203651467), please correct them.

![image](https://user-images.githubusercontent.com/63915024/176980358-2449ca18-d98b-4c6e-a78c-d8a622074c93.png)